examples/time_frequency/source_label_time_frequency.py

```diff
@@ -122,3 +123,45 @@
     )
     ax.set(xlabel="Time (s)", ylabel="Frequency (Hz)", title=f"ITC ({title})")
     fig.colorbar(ax.images[0], ax=axes[ii])
+
+# %%
+
+##############################################################################
+# In the example above, we averaged power across vertices after calculating
+# power because we provided a single label for power calculation and therefore
+# power of all sources within the single label were returned separately. When
+# we provide a list of labels, power is averaged across sources within each
+# label automatically. With a list of labels, averaging is performed before
+# rescaling, so choose a baseline method appropriately.
+
+
+# Get power from multiple labels
+multi_label_power = source_induced_power(
+    epochs,
+    inverse_operator,
+    freqs,
+    labels,
+    baseline=(-0.1, 0),
+    baseline_mode="mean",
+    n_cycles=n_cycles,
+    n_jobs=None,
+    return_plv=False,
+)
+
+# visually compare evoked power in left and right auditory regions
+fig, axes = plt.subplots(ncols=2)
```

Comment:
Generally produces nicer results
```suggestion
fig, axes = plt.subplots(ncols=2, layout="constrained")
```